client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -416,6 +416,46 @@ $mobile-card-max-width: 440px;
 		}
 	}
 
+	.plan-features-2023-grid__plan-highlight {
```

Comment:
Let's stop using highlight" and refactor to "spotlight" pls . Corresponding function can be `SpotlightPlan`. We make use of highlight in labels and other places already.